packages/frontapp/src/app/models/helperFunctions/transformISBNcode.ts

```diff
@@ -0,0 +1,3 @@
+export const transformISBNcode = (code: string) => {
```

Comment:
Please rename this to `normalizeIdentifier`, because 'normalization' is more specific term, and because identifiers include ISBN, but not limited to it, it also might be EAN-13 or something else.